spec/mailers/user_mailer_spec.rb

```diff
@@ -69,6 +84,11 @@
 
     let(:mail) { UserMailer.contact_request(details) }
 
+    it 'is from the default email' do
+      expect(mail.from).to eq [Figaro.env.email_from]
+      expect(mail[:from].display_names).to eq [Figaro.env.email_from]
+    end
+
     it 'sends to the current email' do
       expect(mail.to).to eq [Figaro.env.support_email]
     end
```

Comment:
Should we used a shared example for these to reduce the duplication and to make it easier to make changes if we need to?